src/main/java/AuthenticationController.java

```diff
@@ -6,5 +8,26 @@
  * It delegates the request processing to the AuthenticationService to perform the required actions.
  */
 public class AuthenticationController {
-    // TODO implement class
+
+    private AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    public User createUser(String username, String password) {
+        return authenticationService.createUser(username, password);
+    }
+
+    public boolean signInUser(String username, String password) {
+        return authenticationService.authenticate(username, password);
```

Comment:
```suggestion
        return authenticationService.signInUser(username, password);
```
nit: just for consistency